dali/python/nvidia/dali/external_source.py

```diff
@@ -137,7 +137,7 @@ class _ExternalSourceGroup(object):
 
     def __init__(self, callback, source_desc, is_multioutput, instances=[], *,
                  cuda_stream=None, use_copy_kernel=None, batch=True, parallel=False,
-                 prefetch_queue_depth=None, batch_info=None):
+                 prefetch_queue_depth=None, bytes_per_sample_hint=None, batch_info=None):
```

Comment:
I'd suggest erroring out or at least warning the when this parameter is specified for a non-parallel external source.

Reply:
@klecki has a good point that switching between parallel / non-parallel mode will be extra bothersome with this being an error. That is the case for PES prefetch_queue_depth, commenting it out every time I switch to sequential mode is infuriating as those things can be. :D